src/font/text.cpp

```diff
@@ -974,6 +980,11 @@ int get_max_height(unsigned size, font::family_class fclass, pango_text::FONT_ST
 		.get_max_glyph_height();
 }
 
+float get_line_spacing_factor() {
```

Comment:
This doesn't need to be a function. It can just be an anon constexpr variable. Or if it is a function it should still be constexpr.

Reply:
Sure.

Reply:
I wonder though if I should add something like set_line_spacing(), or if such a thing is really needed.